spec.html

```diff
@@ -13122,6 +13122,7 @@ <h1>
           1. Assert: IsConstructor(_F_) is *false*.
           1. Assert: _F_ is an extensible object that does not have a *"prototype"* own property.
           1. Set _F_.[[Construct]] to the definition specified in <emu-xref href="#sec-ecmascript-function-objects-construct-argumentslist-newtarget"></emu-xref>.
+        1. Else, set _F_.[[Construct]] to the definition specified in <emu-xref href="#sec-built-in-function-objects-construct-argumentslist-newtarget"></emu-xref>.
```

Comment:
```suggestion
        1. Else,
          1. Set _F_.[[Construct]] to the definition specified in <emu-xref href="#sec-built-in-function-objects-construct-argumentslist-newtarget"></emu-xref>.
```
(@ljharb [suggested](https://github.com/tc39/ecma262/pull/2144#discussion_r473334681) that if one arm of an `If` is multi-line, all should be.)

Reply:
@jmdyck
Thanks, I added your suggestion.